src/Workspaces/Core/Portable/Rename/ConflictEngine/RenamedSpansTracker.cs

```diff
@@ -149,7 +149,7 @@ internal async Task<Solution> SimplifyAsync(Solution solution, IEnumerable<Docum
         {
             if (this.IsDocumentChanged(documentId))
             {
-                var document = solution.GetRequiredDocument(documentId);
+                var document = await solution.GetRequiredDocumentAsync(documentId, includeSourceGenerated: true, cancellationToken).ConfigureAwait(false);
```

Comment:
i wouldn't mind a comment just calling out: yes, we expect rename to be able to change some generated docs, so we need to look for that here to get the right spans.